coding_style/README.adoc

```diff
@@ -23,10 +23,14 @@ Rationale:: even when Ansible currently allows names that are not valid identifi
 Making all names valid identifiers will avoid encountering problems in the future. Dictionary keys that are not valid identifiers are also less intuitive to use in Jinja2 (a dot in a dictionary key would be particularly confusing).
 ====
 
+* Use human-readable variable names
 * Use mnemonic and descriptive names and do not shorten more than necessary.
   A pattern `object[_feature]_action` has proven useful as it guarantees a proper sorting in the file system for roles and playbooks.
   Systems support long identifier names, so use them!
 * Avoid numbering roles and playbooks, you'll never know how they'll be used in the future.
+* All tasks should be named and start with a capital letter, this improves readability.
+* All task names should be written in the imperative (e.g. "Ensure service is running"), this communicates the action of the task.
+* Name all plays and task blocks
```

Comment:
May be add a comment to avoid abbreviations in name and/or use capital letter for abbreviations where it cannot be avoided